README.md

```diff
@@ -10,5 +10,5 @@ v2.9+
 
 on debian
 ```bash
-sudo apt-get install libfuse-dev pkg-config
+sudo apt-get install libfuse-dev pkg-config gcc llvm libclang-dev clang
```

Comment:
I discovered these are required too, they probably exist already on the ubuntu CI configuration but they don't on base Debian and probably base Ubuntu.

Reply:
:+1: you're totally right `bindgen` requires `libclang` to build :-\ which is like a little bit of a bummer but not worth sweating over